### DISCUSSION TITLE
autostart not working

### DISCUSSION THREAD

**Discussion by Krypton0024** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#59] [Q&A] [answered]
There is a program in the autorun log and in the registry, but it does not run on startup.

txt file has "AutomaticallyRunThisProgramOnStartup=1" and "StartThisProgramAsAdministrator=1"

How to fix it?

**Comment by HerMajestyDrMona** [comment, marked as answer]
Does it autostart when `StartThisProgramAsAdministrator=0` ?

**Reply by Krypton0024** [reply]
Yes. Thank you

**Reply by HerMajestyDrMona** [reply]
Ok, seems like Windows is limiting programs from starting as administrator on system startup. There's nothing I can do. If you need it, you can try to use the Task Scheduler to setup programs autostart with administrator permissions on system startup.

**Comment by SlavekT** [comment]
I have the same problem with this settings:
AutomaticallyRunThisProgramOnStartup=1
StartThisProgramAsAdministrator=0
What should I try next?

**Reply by HerMajestyDrMona** [reply]
Can you please check if the registry key `Windows11DragAndDropToTaskbarFix` is created in `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Run` ?

**Comment by Void48** [comment]
Not working for me too

**Reply by HerMajestyDrMona** [reply]
Is the registry key `Windows11DragAndDropToTaskbarFix` is created in `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Run` ?

**Reply by Void48** [reply]
> Is the registry key `Windows11DragAndDropToTaskbarFix` is created in `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Run` ?

yep

**Reply by HerMajestyDrMona** [reply]
Then the program correctly creates Autostart. The problem is in your Antivirus or some just another Windows 11 bug. You're not running the program as administrator, correct? `StartThisProgramAsAdministrator=0`

**Reply by Void48** [reply]
> Then the program correctly creates Autostart. The problem is in your Antivirus or some just another Windows 11 bug. You're not running the program as administrator, correct? `StartThisProgramAsAdministrator=0`

nope

**Reply by HerMajestyDrMona** [reply]
Can you see the entry for "Windows 11 Drag & Drop to the Taskbar (Fix)" in the Task Manager -> Startup?
